src/views/GovernanceTaskList/index.test.tsx

```diff
@@ -38,7 +38,10 @@ describe('The Goveranance Task List', () => {
 
   it('displays only the initial governance steps', async done => {
     const mockStore = configureMockStore();
-    const store = mockStore({ systemIntake: { systemIntake: {} } });
+    const store = mockStore({
+      systemIntake: { systemIntake: {} },
```

Comment:
This data structure seems a bit off to me. Why are we nesting an intake inside of an intake?

Also, it looks like this code predates you, so we can ignore this for now if needed.

Reply:
This is a whoopsie from me. If you look just below, I think we did better for the business case by capturing that data in `form` rather than `businessCase`.

Reply:
it's ok. I've made an eng improvements issue for it. It may even have been my own fault.